guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -9,7 +9,16 @@ There are no highlights with Katello {KatelloVersion}.
 [id="katello-upgrade-warnings"]
 == Upgrade Warnings
 
-There are no upgrade warnings with Katello {KatelloVersion}.
+// There are no upgrade warnings with Katello {KatelloVersion}.
+
+=== Port 8443 disabled by default
+
+On {SmartProxyServer}, the deprecated port 8443 has been disabled by default on new installations and upgrades.
+Clients are expected to be using port 443 when communicating through a {SmartProxyServer}.
```

Comment:
```suggestion
Clients should be using port 443 when communicating through a {SmartProxyServer}.
```

Reply:
We should avoid the use of "should" per IBM SG. How about:
```suggestion
Clients now rely on port 443 when communicating through a {SmartProxyServer}.
```